### PR TITLE
master to main in paperless development docs and CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by MaaxGr**
## Proposed change

Fixed development docs. There is no master branch any more.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other (please explain)

## Checklist:

- [x] I have read & agree with the [contributing guidelines](https://github.com/paperless-ngx/paperless-ngx/blob/main/CONTRIBUTING.md).
- [ ] If applicable, I have tested my code for new features & regressions on both mobile & desktop devices, using the latest version of major browsers.
- [ ] If applicable, I have checked that all tests pass, see [documentation](https://paperless-ngx.readthedocs.io/en/latest/extending.html#back-end-development).
- [ ] I have run all `pre-commit` hooks, see [documentation](https://paperless-ngx.readthedocs.io/en/latest/contributing.html#pre-commit-hooks).
- [x] I have made corresponding changes to the documentation as needed.
- [x] I have checked my modifications for any breaking changes.

### CODE REVIEW

**Review by tribut — APPROVED** [review]
Looks good, thanks.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -127,4 +127,4 @@ All team members are notified when mentioned or assigned to a relevant issue or
 
 We are not overly strict with inviting people to the organization. If you have read the [team permissions](#permissions) and think having additional access would enhance your contributions, please reach out to an [admin](#structure) of the team.
 
-The admins occasionally invite contributors directly if we believe having them on a team will accelerate their work.
\ No newline at end of file
+The admins occasionally invite contributors directly if we believe having them on a team will accelerate their work.
```
Comment (shamoon) [inline CONTRIBUTING.md:130]: ?

Comment (tribut) [reply CONTRIBUTING.md:130]: File is currently missing a newline at the end of the file, this adds it.

Github is terrible for these kinds of changes.

Comment (tribut) [reply CONTRIBUTING.md:130]: Not sure what editor @qcasey was using for https://github.com/paperless-ngx/paperless-ngx/commit/e1c6042189bd85be7ba4b86b1878d069e8f030ec, but [we are requesting editors to add it](https://github.com/paperless-ngx/paperless-ngx/blob/19ff8339bed01f2030fda73e0021a603ab5b90cb/.editorconfig#L8) (and we should).

Comment (shamoon) [reply CONTRIBUTING.md:130]: Ah cool thanks. Ok

**Review by shamoon — APPROVED** [review]

**Pull request merged**